File: DEBERES/Deberes en js y ts/05-SWITCHCondicional.ts
/*
Este programa simula el funcionamiento de un cajero de banco.
*/

console.log("\tBienvenido, este programa simula el funcionamiento de un banco\n");
console.log("\nSe empieza teniendo un saldo inicial de 1000 USD.\n");
const saldoInicial = Number(1000);
console.log("Por favor elija: \nIngresar dinero en cuenta -> 1.\nRetirar dinero de la cuenta. -> 2.\nSalir -> 3.");

const opString = prompt("Ingrese el numero de la actividad que desea realizar: ");
const op = Number(opString);



//Ingreso del condicional switch

switch(op){
    case 1: 
        console.log("Digite la cantidad de USD que desea ingresar:\n");
        const usdExtraString = prompt("Digite la cantidad de USD que desea ingresar:");
        const usdExtra = Number(usdExtraString);
        const saldoFinal = usdExtra+saldoInicial;
        console.log("\nDinero en cuenta: " + saldoFinal);
        break;

    case 2:
        console.log("Digite la cantidad de dinero que va a retirar:\n ");
        const usdRetiroString = prompt("Digite la cantidad de dinero que va a retirar: ");
        const usdRetiro = Number(usdRetiroString);
        
            if(usdRetiro > saldoInicial){
                console.log("\nNO posee la cantidad suficiente de dinero.");
            }else{
                console.log("Su saldo restante es: " + (saldoInicial-usdRetiro));
            }
        break;

    case 3:
        break;
    
    default: 
    console.log("Has elegido una opcion no valida.");

}

File: DEBERES/Deberes en mi lenguaje C++/05-SWITCHCondicional.cpp
#include <iostream>
#include <math.h>
#include <conio.h>
using namespace std;

//programa que simula un cajero automatico con saldo inicial de 1000 dolares.

int main () {
	int saldo_inicial=1000, opc;
	float extra, saldo=0, retiro;
	cout<<"\tBienvenido al cajero virtual"<<endl;
	cout<<"\n1. Ingresar dinero en cuenta. ";
	cout<<"\n2. Retirar dinero de la cuenta. ";
	cout<<"\n3. Salir. ";
	cout<<"\nOpcion: "; cin>>opc; cout<<endl;
	
	//Ingreso del condicional switch
	switch(opc){
		case 1: 
			cout<<"Digite la cantidad de dinero a ingresar: "; cin>>extra;
			saldo=saldo_inicial+extra;
			cout<<"\nDinero en cuenta: "<<saldo; break;
			
		case 2:
			cout<<"Digite la cantidad de dinero que va a retirar: "; cin>>retiro;
			if(retiro>saldo_inicial){ cout<<"\nNO posee la cantidad suficiente de dinero."<<endl;
			}
			else{ cout<<"\nSu saldo restante es: "<<saldo_inicial-retiro<<endl;
			} break;
		case 3: break;		
	}
	
	getch();	
	return 0;
}